src/status_im2/contexts/activity_center/events.cljs

```diff
@@ -153,6 +157,74 @@
   [cofx notification]
   (notifications-reconcile cofx [(assoc notification :read true)]))
 
+(defn- update-read-flag
+  [notifications read?]
+  (reduce (fn [acc notification]
+            (conj acc (assoc notification :read read?)))
+          []
+          notifications))
+
+
+;;;; Mark all notifications as read
+
+(rf/defn mark-all-as-read
+  {:events [:activity-center.notifications/mark-all-as-read]}
+  [{:keys [db] :as cofx}]
+  (when-let [undoable-till (get-in db [:activity-center :mark-all-as-read-undoable-till])]
```

Comment:
should updating the db not be in the `on-success` method? 🤔

Reply:
Moved 👍